src/index.html

```diff
@@ -1,13 +1,72 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="attr-req-value">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__monitor"
+        src="./images/imac.jpeg"
+        alt="Image of monitor"
+      >
+
+      <div class="card__information information">
+        <p class="information__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+
+        <p class="information__code">
+          Код товара: 195434
+        </p>
+
+        <div class="information__rating rating">
+          <div class="rating__stars_4">
+            <div class="rating__stars star"></div>
+            <div class="rating__stars star"></div>
+            <div class="rating__stars star"></div>
+            <div class="rating__stars star"></div>
+            <div class="rating__stars star"></div>
+          </div>
+
+          <p class="rating__feedback">
+            Отзывов: 5
+          </p>
+        </div>
+
+        <div class="information__price">
+          <p class="information__price_text">
```

Comment:
fix your naming everywhere
```suggestion
          <p class="information__price-text">
```